etcdserver/api/membership/cluster.go

```diff
@@ -778,6 +796,36 @@ func clusterVersionFromBackend(lg *zap.Logger, be backend.Backend) *semver.Versi
 	return semver.Must(semver.NewVersion(string(vals[0])))
 }
 
+func downgradeInfoFromBackend(lg *zap.Logger, be backend.Backend) *DowngradeInfo {
+	dkey := backendDowngradeKey()
+	if be != nil {
```

Comment:
Can we panic on `be == nil`? So, we can reduce indentation?

Reply:
Move nil check outside `downgradeInfoFromBackend` to reduce indentation.